### PR TITLE
Club: adding custom styles for navigation block

### LINKED ISSUE #6218: club: Styles: Navigation
Implement navigation following the mockups

image

image

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
Club: adding custom styles for the navigation block

#### Related issue(s):
Fixes: https://github.com/Automattic/themes/issues/6218

### CODE REVIEW

**Review by mikachan — COMMENTED** [review]
The responsive menu looks like this for me with these changes, where the items are being cut off on the left side:

image

I think this is partly related to the `width: 100%` on the individual nav items.

**Comment by matiasbenedetto** [comment]
thanks, @mikachan for the review!

It seems to be working fine now:
image

File: club/style.css [hunk 1]
```diff
@@ -172,6 +172,40 @@ a {
 	text-decoration: none;
  }
 
+/*
+ * Navigation
+ */
+ .wp-block-navigation__responsive-container.is-menu-open .wp-block-navigation__responsive-container-content .wp-block-navigation__container {
+	align-items: flex-start;
```
Comment (mikachan) [inline club/style.css]: Actually, now I've taken another look at this, I don't think we need any of these `flex-start`s (lines 179, 183 and 197). I think we can change the alignment on the navigation block itself (`justifyContent`) to left aligned, and as long as the row container block is set to space between (which it currently is), the menu will appear aligned to the right on larger resolutions. e.g. change the navigation block to something like this:

```
<!-- wp:navigation {"overlayBackgroundColor":"background","overlayTextColor":"foreground","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left","orientation":"horizontal"},"style":{"spacing":{"margin":{"top":"0"}}}} /-->
```

Comment (matiasbenedetto) [reply club/style.css]: Thanks for noticing this @mikachan. I wasn't aware of the ability of the navigation block to be configured in such a way. I removed that CSS and replaced it with the block configuration.

**Comment by pbking** [comment]
🤔 The latest of this is showing a white background on the responsive navigation for me:

image

File: club/parts/header.html [hunk 2]
```diff
@@ -8,6 +8,8 @@
 	<!-- /wp:group --></div>
 	<!-- /wp:group -->
 	
-	<!-- wp:navigation {"__unstableLocation":"primary","overlayBackgroundColor":"background","overlayTextColor":"foreground","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"right","orientation":"horizontal"},"style":{"spacing":{"margin":{"top":"0"}}}} /--></div>
+	<!-- wp:navigation {__unstableLocation":"primary","overlayBackgroundColor":"background","overlayTextColor":"foreground","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left","orientation":"horizontal"},"style":{"spacing":{"margin":{"top":"0"}}}} /--></div>
```
Comment (mikachan) [inline club/parts/header.html]: It looks like there's a missing `"` from the beginning of "__unstableLocation".

Comment (matiasbenedetto) [reply club/parts/header.html]: Yep, thanks for notice that!

**Comment by matiasbenedetto** [comment]
It should be working like this:

Desktop:
image


Mobile:
image
image

**Review by mikachan — APPROVED** [review]
LGTM! 🚢

**Pull request merged**